sunpy/net/helio/hec.py

```diff
@@ -174,14 +181,14 @@ def select_table(self):
         table_list.sort()
         for index, table in enumerate(table_list):
             print(f'{index + 1} - {table.decode()}')
-
         while True:
             user_input = input(f"\nPlease enter a table number between 1 and {len(table_list)} "
                                "('e' to exit): ")
             if user_input.lower() == "e" or user_input.lower() == "exit":
                 return None
             if user_input.isdigit() and 1 <= int(user_input) <= len(table_list):
                 table_no = int(user_input)
-                return table_list[table_no - 1]
+                table = table_list[table_no - 1]
+                return table
```

Comment:
Why?

Reply:
I was working on something else and undid it but forgot to undo this.